packages/@aws-cdk/aws-codebuild/package.json

```diff
@@ -129,10 +129,12 @@
       "docs-public-apis:@aws-cdk/aws-codebuild.Cache.none",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2_2",
+      "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2_3",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.AMAZON_LINUX_2_ARM",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.STANDARD_1_0",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.STANDARD_2_0",
       "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.STANDARD_3_0",
+      "docs-public-apis:@aws-cdk/aws-codebuild.LinuxBuildImage.STANDARD_4_0",
```

Comment:
Can you add a comment instead of this exclusion?

```ts
/** The `aws/codebuild/standard:4.0` build image. */
public static readonly STANDARD_4_0 = LinuxBuildImage.codeBuildImage('aws/codebuild/standard:4.0');
```

Reply:
@skinny85 no problem this has been done.